.github/test-spec.yml

```diff
@@ -1,4 +1,17 @@
 # This is the Jenkins ci variant of the .github/labler.yaml
+
+"CI-run-zephyr-twister":
+  - any:
+      - "!.github/**/*"
+      - "!.vscode/**/*"
+      - "!doc/**/*"
+      - "!test-manifests/**/*"
+      - "!CODEOWNERS"
+      - "!LICENSE"
+      - "!**/*.rst"
+      - "!VERSION"
+      - "!ncs_version.h.in"
+
```

Comment:
add more? 

!submanifests/**/*
!scripts/**/*
!snippets/**/*
!scripts/**/*
!CODEOWNERS
!MAINTAINERS.yml
!version.h.in
!Jenkinsfile
"!**/*.md"

Reply:
IMO we can leave the scripts to be traced (not excluded) for now. Later on we can play with a smarter scope selection. E.g. since recently, twister vastly improved on providing unit tests for itself and those are triggered as a github workflow,. However, I think many of those haven't came to sdk-zephyr yet. Also, the test_plan script which we will use in sdk-nrf has means of descoping certain patterns from being added to the scope. If we expand it somehow to reliably work with changes in zephyr we could leave the decision to be made by that script

Reply:
why snippets are excluded?